### DISCUSSION TITLE
Choose conftest.py file to run particular test

### DISCUSSION THREAD

**Discussion by andrsGutirrz** [pytest-dev/pytest#10620] [Q&A] [answered]
Hello, 
I have a suite of tests in which I want to select which `conftest.py` file I want to use. 
How can I select a `conftest.py` file to run my tests?  
Here is a  screenshot for more context: 
<img width="255" alt="image" src="https://user-images.githubusercontent.com/16330583/209984321-1098ea7b-6df2-45ff-ad4c-6761f4214bda.png">

How can I target conftest.py at root level, instead of /test level?

**Comment by andrsGutirrz** [comment]
I have tried with `-c` and  `--confcutdir=dir` flags but they did not work

**Comment by RonnyPfannschmidt** [comment, marked as answer]
All conftest files in parent directories apply to particular tests,

With confcutdir one can exclude some parent folders, but never the other way around
